mythtv/libs/libmythui/mythmainwindow.cpp

```diff
@@ -775,6 +771,10 @@ void MythMainWindow::Init(bool MayReInit)
         MythNotification notification(warningmsg, "");
         m_priv->m_nc->Queue(notification);
     }
+    
```

Comment:
I think these four lines can go, if the calls are not needed anymore then they do not need to remain as comment.

Reply:
I did intend to mark these lines for removal:
    
    // from ReinitDone
    //ShowPainterWindow();
    //MoveResize(m_screenRect);